src/components/Switch/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class Switch extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            chosenParamIndex: 0,
+        };
+    }
+    handleClick(which) {
+        this.setState({chosenParamIndex: which })
+    }
+
+    getClass(index) {
+        return "Switch__Param " + (this.state.chosenParamIndex === index ? "Switch_activeParam" : "");
+    }
+
+    render() {
+        const { params, text } = this.props;
+        return (
+            <div className="Switch">
+                {text}
+                <span className={this.getClass(0)} onClick={() => this.handleClick(0)}>
```

Comment:
it's better to bind handleClick in the constructor once to avoid unnecessary anonymous function creation each time component is rendered.

[https://reactjs.org/docs/handling-events.html](https://reactjs.org/docs/handling-events.html)

Reply:
fixed